### PR TITLE
Prius Prime differentiated from Prius

### PR DESCRIPTION
**Pull request by ErichMoraga**
The need to treat the Prime separately from the Prius has become increasingly evident.  I've tested this code (3 files changed) with several users (various makes/models) successfully.

### CODE REVIEW

**Comment by rbiasini** [comment]
Changes look good. I only recommend to change `CAR.PRIME` into `CAR.PRIUS_PRIME`
However, before merging these changes I would need some evidence (or at least be able to verify it) that the 2 car models behave differently. Can you PM on slack the list of dongle ids you tested?

**Comment by ErichMoraga** [comment]
I've PM'd you my Dongle ID, and put out an alert/request in #toyota for others to PM you too.  I also agree w/ the "CAR.PRIME into CAR.PRIUS_PRIME" suggestion.

**Review by ErichMoraga — COMMENTED** [review]
Changed CAR.PRIME to CAR.PRIUS_PRIME

**Comment by rbiasini** [comment]
build failed @ErichMoraga

File: selfdrive/car/toyota/values.py [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 class CAR:
   PRIUS = "TOYOTA PRIUS 2017"
+  PRIME = "TOYOTA PRIUS PRIME 2017"
```
Comment (rbiasini) [inline selfdrive/car/toyota/values.py]: change to PRIUS_PRIME

Comment (ErichMoraga) [reply selfdrive/car/toyota/values.py]: Oh duh!  Getting on it...

**Review by ghost — APPROVED** [review]

**Comment by Glidepathuav** [comment]
Thank you !

**Review by ghost — APPROVED** [review]

**Comment by pianohawk** [comment]
I have a 2017 Prius Prime and am interested in helping anyway I can.  Is openpilot ready for this vehicle?

**Comment by andy867** [comment]
@pianohawk 
Most definitely. @Erich Moraga and @LarryJustin have been working hard to get the prime to be one of the most supported vehicles for openpilot.

**Comment by mtourangeau** [comment]
Works fine. Little pingpong but getting better.

**Comment by pianohawk** [comment]
OMG!!!!  If I order the hardware, I can run it as soon as it arrives?

**Comment by mtourangeau** [comment]
https://community.comma.ai/wiki/index.php/Installing_OpenPilot_on_the_EON

Remember to set the jumper correctly. That will be your major problem.

File: selfdrive/car/toyota/interface.py [hunk 2]
```diff
@@ -91,7 +91,7 @@ def get_params(candidate, fingerprint):
       ret.steerKpV, ret.steerKiV = [[0.45], [0.015]]
       ret.steerKf = 0.0000675   # full torque for 10 deg at 80mph means 0.00007818594
       # TODO: Prius seem to have very laggy actuators. Understand if it is lag or hysteresis
-      ret.steerActuatorDelay = 0.40
+      ret.steerActuatorDelay = 0.43
```
Comment (energee) [inline selfdrive/car/toyota/interface.py]: Damn, that thing is slow as fuck :) I think @pd0wm has a way to measure exact delay

Comment (ErichMoraga) [reply selfdrive/car/toyota/interface.py]: Doing a lot of tests, so not necessarily a final value.

**Comment by mtourangeau** [comment]
I’m testing those value and it’s much better. We can almost close #300.

**Pull request closed without merging**